setup.rst

```diff
@@ -65,7 +65,7 @@ Symfony application using Composer:
 .. code-block:: terminal
 
     # run this if you are building a traditional web application
-    $ composer create-project symfony/skeleton:"^5.4" my_project_directory
+    $ composer create-project symfony/website-skeleton:"^5.4" my_project_directory
```

Comment:
Here it is not necessary to do `composer create-project symfony/website-skeleton` because it is identical to `composer require webapp` (the command that is indicated after).